examples/websockets/client.js

```diff
@@ -0,0 +1,18 @@
+var sharedb = require("sharedb/lib/client");
+
+var connection = new sharedb.Connection(new WebSocket("ws://localhost:8888"));
+
+var doc = connection.get("collection", "document");
+
+doc.subscribe(function(error){
+  if (!doc.type) {
+    doc.create({x: "hello"});
+  }
+
+  if (doc.type) {
+    // `doc` ready, make ops
+    setTimeout(function() { // timeout need for example (to delay after create)
```

Comment:
When I tried this, this branch didn't hit in the first connection (the first pageload), so I don't think the timeout is necessary.